aten/src/ATen/native/native_functions.yaml

```diff
@@ -9490,31 +9490,31 @@
     CPU: slow_conv_transpose3d_backward_cpu
     CUDA: slow_conv_transpose3d_backward_cuda
 
-- func: thnn_conv2d.out(Tensor self, Tensor weight, int[2] kernel_size, Tensor? bias=None, int[2] stride=1, int[2] padding=0, *, Tensor(a!) out) -> Tensor(a!)
+- func: _slow_conv2d.out(Tensor self, Tensor weight, int[2] kernel_size, Tensor? bias=None, int[2] stride=1, int[2] padding=0, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
I've restored the name of the top-level function but I think changing the names of `_forward` and `_backward` is okay since the signature is changing either way.